broker/configuration.go

```diff
@@ -24,11 +24,12 @@ import (
 
 // Configuration represents configuration of Kafka broker
 type Configuration struct {
-	Address      string     `mapstructure:"address" toml:"address"`
-	Topic        string     `mapstructure:"topic" toml:"topic"`
-	PublishTopic string     `mapstructure:"publish_topic" toml:"publish_topic"`
-	Group        string     `mapstructure:"group" toml:"group"`
-	Enabled      bool       `mapstructure:"enabled" toml:"enabled"`
-	OrgWhitelist mapset.Set `mapstructure:"org_white_list" toml:"org_white_list"`
-	SaveOffset   bool       `mapstructure:"save_offset" toml:"save_offset"`
+	Address             string     `mapstructure:"address" toml:"address"`
+	Topic               string     `mapstructure:"topic" toml:"topic"`
+	PublishTopic        string     `mapstructure:"publish_topic" toml:"publish_topic"`
+	Group               string     `mapstructure:"group" toml:"group"`
+	Enabled             bool       `mapstructure:"enabled" toml:"enabled"`
+	OrgWhitelist        mapset.Set `mapstructure:"org_whitelist" toml:"org_whitelist"`
```

Comment:
Why do you make `toml:"org_whitelist"`` but in the config you use `org_whitelist_file`?

Reply:
@quarckster thanks, last place I missed....

Reply:
@quarckster  this is actually not wrong, because this is broker/ configuration and the OrgWhitelist is not loaded automatically, but by calling `config.Broker.OrgWhitelist = getOrganizationWhitelist()`, so it will still work

but I'm fixing it

Reply:
done